src/components/web-monitoring-ui.tsx

```diff
@@ -57,3 +71,18 @@ export default class WebMonitoringUi extends React.Component<undefined, IWebMoni
         return {api};
     }
 }
+
+function getPagesByUser (userName: string): Promise<any> {
```

Comment:
**A2**: Makes sense to abstract this. But don't think we should have a module for UI API. These functions look like they would still be needed to interact with DB eventually. Can we move them there and re-implement when switch happens? Mark them as `tempfix` in comments?